plugins/kn-admin/README.md

```diff
@@ -0,0 +1,66 @@
+# kn-admin
+A plugin for kn client for management
```

Comment:
Change to “A kn plugin for Knative cluster management”

Reply:
Fixed!